integrationTests/chainSimulator/bridge/simpleEsdt_test.go

```diff
@@ -0,0 +1,446 @@
+package bridge
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/stretchr/testify/require"
+
+	"github.com/multiversx/mx-chain-go/config"
+	chainSim "github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/dtos"
+)
+
+const (
+	simpleEsdtTestWasmPath = "testdata/simple-esdt-test.wasm"
+	creatorAddress         = "8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8"
+	burnEndpoint           = "burn"
+)
+
+func TestSovereignChainSimulator_CreateNftWithManyQuantityShouldFail(t *testing.T) {
+	if testing.Short() {
```

Comment:
Should we add some t.Parallel in these tests?

Reply:
I see `t.Parallel()` is not used in chain simulator tests, I'm not sure why. But added parallel for the tests in this file

Reply:
The parallel execution doesn't work for chain simulator tests. Mihai Iuga said there are some port issues if tests are running in parallel